src/templates/static/legal/complaints.jsx

```diff
@@ -3,7 +3,10 @@ import React from 'react';
 const Complaints = () => (
     <div>
         <h2 data-anchor='complaints-and-disputes'>{it.L('Complaints and disputes')}</h2>
-        <p>{it.L('If the client would like to file a complaint with regards to the Company\'s service, the client can contact the Company at [_1], providing any relevant details relating to the client\'s complaint. The Company will investigate each inquiry and provide a final response in the shortest time possible.', '<a href=\'mailto:complaints@binary.com\'>complaints@binary.com</a>')}</p>
+        
+        <p data-show='-malta'>{it.L('If the client would like to file a complaint with regards to the Company\'s service, the client can contact the Company at [_1], providing any relevant details relating to the client\'s complaint. The Company will investigate each inquiry and provide a final response in the shortest time possible.', '<a href=\'mailto:complaints@binary.com\'>complaints@binary.com</a>')}</p>
+        <p data-show='malta'>{it.L('If the client would like to file a complaint with regards to the Company\'s service, the client can contact the Company at [_1], providing any relevant details relating to the client\'s complaint. If you are registered with Deriv (Europe) Limited, you can also place a complaint by calling [_2]. The Company will investigate each inquiry and provide a final response in the shortest time possible.', '<a href=\'mailto:complaints@binary.com\'>complaints@binary.com</a>','<a href=\'call:+447723580049\'>+447723580049</a>')}</p>
```

Comment:
IMO it's better to show the common string and only have a span wrapper around the unshared string. then translators don't need to translate the same long sentence that is shared between these twice.

Reply:
Thank you for the tip. Fixed.